web-app/.storybook/webpack.config.js

```diff
@@ -12,27 +12,16 @@ module.exports = ({ config }) => {
   config.module.rules.push({
     test: /\.(ts|tsx)$/,
     loader: require.resolve('babel-loader'),
-    options: {
-      plugins: [
-        new MiniCssExtractPlugin(),
-        [
-          'babel-plugin-import',
-          {
-            libraryName: '@alifd/next',
-            style: true,
-          },
-        ],
-      ],
-      presets: [
-        // react-app
-        ['react-app', { flow: false, typescript: true }],
-        // allow emotion css prop on html
-        ['@emotion/babel-preset-css-prop'],
-      ],
-    },
   })
 
   config.plugins.push(new MiniCssExtractPlugin({ filename: '[name].css' }))
+  // config.plugins.push([
+  //   'babel-plugin-import',
+  //   {
+  //     libraryName: '@alifd/next',
+  //     style: true,
+  //   },
+  // ])
```

Comment:
I worry some components may not work without @alifd/next and emotion support. But I may be wrong!

Reply:
I've updated my branch. I've also enabled the plugin. Seems to be working.